KlarnaOnDemand/KODWebViewController.m

```diff
@@ -121,5 +115,30 @@ - (void)unregisterJockeyCallbacks {
   [Jockey off:JockeyUserError];
 }
 
+- (void)webView:(WKWebView *)webView didFinishNavigation:(WKNavigation *)navigation {
```

Comment:
You might want to move all the WKNavigationDelegate to one place, maybe even under a pragma mark